doc/developer_guide/developer_guide.md

```diff
@@ -28,50 +28,86 @@ bash install.sh
-- `export-vm` - creates a new VM image from a running EC2-Instance
-- `install-dependencies` - starts an ansible-installation onto an existing EC-2 instance
-- `reset-password` - resets password on a remote EC-2-instance via ansible
-- `setup-ec2` - starts a new EC2 instance (based on an Ubuntu AMI)
-- `setup-ec2-and-install-dependencies` - starts a new EC2 instance and install dependencies via Ansible
-- `show-aws-assets` - shows AWS entities associated with a specific keyword (called __asset-id__)
-- `start-test-release` - starts a Test Release flow
-- `make-ami-public` - Changes permissions of an existing AMI such that it becomes public
+* `export-vm`: Create a new VM image from a running EC2-Instance.
+* `install-dependencies`: Start an ansible-installation onto an existing EC-2 instance.
+* `reset-password`: Reset password on a remote EC-2-instance via ansible.
+* `setup-ec2`: Start a new EC2 instance (based on an Ubuntu AMI).
+* `setup-ec2-and-install-dependencies`: Start a new EC2 instance and install dependencies via Ansible.
+  * The script will print the required SSH login for manual inspection or interaction with the EC2 instance.
+  * The instance is kept running until the user presses Ctrl-C.
+* `show-aws-assets`: Show AWS entities associated with a specific keyword (called __asset-id__).
+* `start-test-release-build`: Start a Test Release flow.
+* `make-ami-public`: Change permissions of an existing AMI such that it becomes public.
+
+Script `start-test-release-build` requires environment variable `GH_TOKEN` to contain a valid token for access to Github.
 
 ### Deployment commands
 
 The following commands can be used to deploy the infrastructure onto a given AWS account:
-- `setup-ci-codebuild` - deploys the AWS Codebuild cloudformation stack which will run the ci-test
-- `setup-vm-bucket` - deploys the AWS Bucket cloudformation stack which will be used to deploy the VM images
-- `setup-release-codebuild` - deploys the AWS Codebuild cloudformation stack which will be used for the release-build
-- `setup-vm-bucket-waf` - deploys the AWS Codebuild cloudformation stack which contains the WAF Acl configuration for the Cloudfront distribution of the VM Bucket
-- `create-docker-image` - creates a Docker image for data-science-sandbox and deploys it to hub.docker.com/exasol/data-science-sandbox
+* `setup-ci-codebuild`: Deploy the AWS Codebuild cloudformation stack which will run the ci-test.
+* `setup-vm-bucket`: Deploy the AWS Bucket cloudformation stack which will be used to deploy the VM images.
+* `setup-release-codebuild`: Deploy the AWS Codebuild cloudformation stack which will be used for the release-build.
+* `setup-vm-bucket-waf`: Deploy the AWS Codebuild cloudformation stack which contains the WAF Acl configuration for the Cloudfront distribution of the VM Bucket.
+* `create-docker-image`: Create a Docker image for data-science-sandbox and deploy it to hub.docker.com/exasol/data-science-sandbox.
```

Comment:
Not a deployment command, but a release command

Reply:
see next push